<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>451</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Welcome!</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To begin, please select Create &gt;&gt; Project. </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Attr</string>
     </attribute>
     <widget class="QLabel" name="ID">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Measure:</string>
      </property>
     </widget>
     <widget class="QLabel" name="commit_confirm">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="llayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>66</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="LabelLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="src">
         <property name="text">
          <string>Source:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="stim">
         <property name="text">
          <string>Stimulus:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="env">
         <property name="text">
          <string>Environment:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="afct">
         <property name="text">
          <string>Artifact(s):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rsp">
         <property name="text">
          <string>Response:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rspm">
         <property name="text">
          <string>Response</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aclayoutWidget">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>430</y>
        <width>158</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="ActionLayout">
       <item>
        <widget class="QPushButton" name="destroyComplete">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Destroy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addComplete">
         <property name="text">
          <string>Commit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="attID">
         <property name="text">
          <string>Attribute ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="idLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>258</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="inputLayout">
       <item>
        <widget class="QPlainTextEdit" name="srced"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="stimed"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="enved"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="afcted"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="rsped"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="rspmed"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>141</width>
      <height>471</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Project</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionProject"/>
    <addaction name="actionAttribute"/>
   </widget>
   <widget class="QMenu" name="menuDestroy">
    <property name="title">
     <string>Destroy</string>
    </property>
    <addaction name="destroyproj"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPrint_to_File"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCreate"/>
   <addaction name="menuDestroy"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionProject">
   <property name="text">
    <string>Project</string>
   </property>
  </action>
  <action name="actionAttribute">
   <property name="text">
    <string>Attribute</string>
   </property>
  </action>
  <action name="destroyproj">
   <property name="text">
    <string>Project</string>
   </property>
  </action>
  <action name="destroyattr">
   <property name="text">
    <string>Attribute</string>
   </property>
  </action>
  <action name="destroycat">
   <property name="text">
    <string>Category</string>
   </property>
  </action>
  <action name="actionChange_Project_Name">
   <property name="text">
    <string>Change Project Name</string>
   </property>
  </action>
  <action name="actionPrint_to_File">
   <property name="text">
    <string>Print to File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionAttribute</sender>
   <signal>triggered()</signal>
   <receiver>tabBar</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addComplete</sender>
   <signal>clicked()</signal>
   <receiver>destroyComplete</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
